src/backend/distributed/planner/multi_planner_utils.c

```diff
@@ -0,0 +1,863 @@
+/*
+ * multi_planner_utils.c
+ *
+ * This file contains functions helper functions for planning
+ * queries with colocated tables and subqueries.
+ *
+ * Copyright (c) 2017-2017, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+#include "postgres.h"
+#include "c.h"
```

Comment:
Don't ever include c.h yourself when postgres.h is also included. Yes, we do that in some places (where I didn't pay attention), but that shouldn't spread around.